packages/kbn-interpreter/.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["@kbn/babel-preset/webpack_preset"],
+  "presets": ["@kbn/babel-preset/node_preset"],
```

Comment:
Currently being imported in both server/client. Will need to create separate builds, in the meantime commonjs should suffice.